Lib/ctypes/_util.py

```diff
@@ -0,0 +1,15 @@
+from sys import maxsize
```

Comment:
I don't think that a single function justify the creation of a whole module. Just put this code into the existing util.py

Reply:
Why not putting this code in util.py directly? If it's really specific to AIX, maybe put it in _aix.py instead?